flax/linen/__init__.py

```diff
@@ -23,10 +23,19 @@
   elu as elu,
   gelu as gelu,
   glu as glu,
+  hard_sigmoid as hard_sigmoid,
+  hard_silu as hard_silu,
+  hard_swish as hard_swish,
+  hard_tanh as hard_tanh,
   leaky_relu as leaky_relu,
   log_sigmoid as log_sigmoid,
   log_softmax as log_softmax,
+  logsumexp as logsumexp,
+  normalize as normalize,
```

Comment:
could we add the `standardize` alias for `normalize` as well? (this name change follows stats literature naming)

Reply:
I initially added it, but then removed it again when I saw that it had only been added to JAX very recently (not in version 0.3, which I understand to be Flax's dependency).

I guess we could do something like `try: import standardize; except: pass`, but this might be bad style. Or we could define the alias ourselves.

Reply:
What do you think would be the best option here?

Reply:
It seems cleaner to create an issue to add this once JAX rolls out a new version.

Reply:
Added #2119